src/main/resources/templates/email.twig

```diff
@@ -0,0 +1,303 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Validatar Report</title>
+    <style>
+        /* modified from http://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.css */
```

Comment:
Link to this instead? https://github.com/twbs/bootstrap/blob/v3.3.7/dist/css/bootstrap.css